tests/test_formats.py

```diff
@@ -1060,6 +1062,511 @@ def test_format_number_with_sep_dec_marks():
+    "currency,use_subunits,decimals,drop_trailing_dec_mark,use_seps,placement,incl_space,x_out",
+    [
+        (None, True, None, True, True, "left", False, ["$1,234,567.00", "−$5,432.37"]),
+        ("USD", True, None, True, True, "left", False, ["$1,234,567.00", "−$5,432.37"]),
+        ("EUR", True, None, True, True, "left", False, ["&#8364;1,234,567.00", "−&#8364;5,432.37"]),
+        (None, False, None, True, True, "left", False, ["$1,234,567", "−$5,432"]),
+        (None, False, 4, True, True, "left", False, ["$1,234,567.0000", "−$5,432.3700"]),
+        (None, True, 4, True, True, "left", False, ["$1,234,567.0000", "−$5,432.3700"]),
+        (None, True, 0, False, True, "left", False, ["$1,234,567.", "−$5,432."]),
+        (None, True, None, True, False, "left", False, ["$1234567.00", "−$5432.37"]),
+        (None, True, None, True, True, "right", False, ["1,234,567.00$", "−5,432.37$"]),
+        (None, True, None, True, True, "right", True, ["1,234,567.00 $", "−5,432.37 $"]),
+        (None, True, None, True, True, "left", True, ["$ 1,234,567.00", "−$ 5,432.37"]),
+    ],
+)
+def test_fmt_currency(
+    currency: Optional[str],
+    use_subunits: bool,
+    decimals: Optional[int],
+    drop_trailing_dec_mark: bool,
+    use_seps: bool,
+    placement: str,
+    incl_space: bool,
+    x_out: str,
+):
+
+    df = pd.DataFrame({"x": [1234567, -5432.37]})
+
+    gt = GT(df).fmt_currency(
+        columns="x",
+        currency=currency,
+        use_subunits=use_subunits,
+        decimals=decimals,
+        drop_trailing_dec_mark=drop_trailing_dec_mark,
+        use_seps=use_seps,
+        placement=placement,
+        incl_space=incl_space,
+    )
+    x = _get_column_of_values(gt, column_name="x", context="html")
+    assert x == x_out
+
+
+@pytest.mark.parametrize(
+    "force_sign,x_out",
+    [
+        (
+            False,
+            [
+                "−$234.65",
+                "−$0.00",
+                "−$0.10",
+                "$0.00",
+                "$2,352.23",
+                "$12,354.30",
+                "$9,939,293,923.23",
+            ],
+        ),
+        (
+            True,
+            [
+                "−$234.65",
+                "−$0.00",
+                "−$0.10",
+                "$0.00",
+                "+$2,352.23",
+                "+$12,354.30",
+                "+$9,939,293,923.23",
+            ],
+        ),
+    ],
+)
+def test_fmt_currency_force_sign(force_sign: bool, x_out: str):
+    df = pd.DataFrame({"x": [-234.654, -0.000634, -0.1, 0, 2352.23, 12354.3, 9939293923.23]})
+
+    gt = GT(df).fmt_currency(columns="x", force_sign=force_sign)
+    x = _get_column_of_values(gt, column_name="x", context="html")
+    assert x == x_out
+
+
+# ------------------------------------------------------------------------------
+# Test `fmt_time()`
+# ------------------------------------------------------------------------------
+
+
+@pytest.mark.parametrize(
+    "time_style,x_out",
```

Comment:
Since each case is run against the same 7 input times, it's not clear which of these inputs tests something new per timestyle, and which is redundant.

Rather than each case involving a time_style and a the same list of 7 inputs, can you make each case a time_style, 1 input, and the output expected? A case should only be added if it tests some new bit underlying logic.